### PR TITLE
fix: Use platform HTTP server in REST DSL example

### PR DESCRIPTION
**Pull request by astefanutti**
**Release Note**
```release-note
NONE
```

### CODE REVIEW

File: examples/routes-rest.js [hunk 1]
```diff
@@ -17,13 +17,9 @@
 //
 // To run this integrations use:
 //
-//     kamel run --name=withrest --dependency=camel-undertow examples/routes-rest.js
+//     kamel run --name=withrest
```
Comment (nicolaferraro) [inline examples/routes-rest.js:20]: Maybe this does not work ;)

Comment (astefanutti) [reply examples/routes-rest.js:20]: It seems not to be working with the Undertow dependency, as two servers get started. I'm catching up though :)

Comment (astefanutti) [reply examples/routes-rest.js:20]: Ah damn, I get your point :D

Comment (astefanutti) [reply examples/routes-rest.js]: It should be better now :D

**Pull request merged**